problems/arraysIntro.js

```diff
@@ -4,57 +4,85 @@
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToEnd() {}
+function addFruitToEnd (fruits, fruit) {
+  fruits.push(fruit);
+  return fruits;
+
+
+} 
+ 
 
 /**
  * Adds a fruit to the front of an array of fruits and returns the array.
  * @param {string[]} fruits
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToFront() {}
+function addFruitToFront (fruits, fruit) {
+  fruits.unshift(fruit)
+  return fruits;
+}
+
 
 /**
  * Removes the last fruit from an array of fruits and returns the array.
  * @param {string[]} fruits
  * @returns {string[]}
  */
-function removeLast() {}
+function removeLast(fruits) {
+  fruits.pop(fruits);
```

Comment:
We don't need to put the array inside pf the [pop method](https://github.com/joinpursuit/Pursuit-Core-Web/blob/master/fundamentals/arrays/README.md#pop)